stellar/utils.js

```diff
@@ -164,7 +164,7 @@ async function sendTransaction(tx, server, action, options = {}) {
 }
 
 async function broadcast(operation, wallet, chain, action, options = {}, simulateTransaction = false) {
-    const server = new rpc.Server(chain.rpc);
+    const server = new rpc.Server(chain.rpc, { allowHttp: true });
```

Comment:
this workaround is being applied for non-local networks as well. using http instead of https is insecure for production. condition this on the network being local